props/detonator.h

```diff
@@ -134,3 +138,174 @@ class Detonator : public PROP_INHERIT_PREFIX PropBase {
 };
 
 #endif
+
+/************************************\
+|*                                  *|
+|*   DETONATOR DISPLAY CONTROLLER   *|    // added by OlivierFlying747-8
+|*                                  *|
+\************************************/
+
+// If you want to add more, replace all the "***xxxn***, ***XXXn***, ***Xxxn***" with appropriate
+// effects name(s), and respect the CAPS, no caps & Some Caps. Or else it may not compile or work!
+
+#ifdef PROP_BOTTOM
+
+#define ONCE_PER_DETONATOR_EFFECT(X)  \
+  X(bgnarm)                           \
+  X(endarm)                           \
+  X(boom) // Add a backslash to this line when adding the next line.
+          // The last line in the list needs no backslash at the end!
+//X(***xxx1***)
+//X(***xxx2***)
+//X(***xxx3***)
+
+#ifdef INCLUDE_SSD1306
+
+struct DetonatorDisplayConfigFile : public ConfigFile {
```

Comment:
So, is anybody actually going to use this?
A detonator with a display seems weird.
I'm not totally against adding it, but adding something that nobody will use is generally... useless?

Reply:
Very well, but please remember: no untested code....